src/compo/modis_aod2ioda.py

```diff
@@ -129,9 +130,7 @@ def _read(self):
             sen_zen = sen_zen[pos_index]
             unc_land = unc_land[pos_index] * 1E-3  # see scale factor
             modis_time = modis_time[pos_index]
-            obs_time = np.full(len(modis_time), long_missing_value, dtype=np.int64)
-            for n, t in enumerate(modis_time):
-                obs_time[n] = round(t)
+            obs_time = np.full(len(modis_time), np.around(modis_time), dtype=np.int64)
```

Comment:
@jeromebarre this look correct this is a replacement for the two preceding lines

I think the assumption here is the `modis_time` is just a single scalar?

Reply:
oh no I see now... it's where `aod` is defined

Reply:
regardless line 133 should provide the same behavior as before the issue is if a valid `pos_index` where `aod` is greater than 0 may have a missing time which will not get set.  happy to punt on this for now?